packages/core/src/lib/relay/index.ts

```diff
@@ -30,7 +30,7 @@ export type Observer<T extends IDecodedMessage> = {
   callback: Callback<T>;
 };
 
-export type CreateOptions = {
+export interface RelayCreateOptions extends GossipsubOpts {
```

Comment:
are these changes under the scope of the titled PR?

Reply:
Yes so typedoc does not handle `type`s very well especially cross repo, hence the switch to an interface.

In terms of transpiled JS code, I believe one or the other does not matter much so opted for the one typedoc is happy with.